{"uuid": "e7cb80dd-d9ae-40ed-9f5f-aa41397641ff", "children": ["909d00b4-cd7c-46c4-8c59-dd188dc8317c", "09ccd7cd-6e1d-4b68-a8f9-cfef9148deb0", "359151d3-6d42-4882-9509-cc012556cf5b", "f6269333-6c85-45de-a95d-3feddefbcaf6", "fb83b9ba-5347-43c7-92ec-f9d5e578427d", "92fd60a9-a799-4c12-9e58-201d36f393be"], "befores": [{"name": "set_up", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'Login_page' has no attribute 'login_url'. Did you mean: 'login_as'?\n", "trace": "  File \"C:\\Users\\tan\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\test-package-xh99BTJe-py3.12\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\tan\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\test-package-xh99BTJe-py3.12\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\tan\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\test-package-xh99BTJe-py3.12\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\tan\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\test-package-xh99BTJe-py3.12\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\tan\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\test-package-xh99BTJe-py3.12\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\QA-practice\\Saucelabs\\python\\tests\\test_login.py\", line 31, in set_up\n    base_url = Login_page.login_url\n               ^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1740990567510, "stop": 1740990567510}], "start": 1740990567510, "stop": 1740990567666}